Sources_v3/Models/User.swift

```diff
@@ -34,6 +34,9 @@ public class _ChatUser<ExtraData: UserExtraData> {
     /// An indicator whether the user is banned.
     public let isBanned: Bool
     
+    /// An indicator whether the user is flagged by the current user.
+    public let isFlaggedByCurrentUser: Bool
+    
```

Comment:
I like this API! Let's maybe add one more thing to the documentation comment:
```
/// - Note: Please be aware that the value of this field is not persisted on the server, and is valid only locally for the current session.
```

Reply:
fixed in 40e3bc6e8e720ed9c958e2bc39e3a4f705990696